.github/workflows/release.yml

```diff
@@ -1,17 +1,17 @@
 name: Release
 on:
   push:
-    tags: ["v*"]
-
+    branches: [ master ]
+  release:
+    types: [ published ]
```

Comment:
I remember having issues with `tags: ["v*"]`, either it required some weird permissions or some tokens weren't available or something... But `types: [ published ]` fixed it for good.